plugins/versionpress/src/Utils/RequirementsChecker.php

```diff
@@ -274,7 +274,7 @@ private function tryWrite()
 
     private function tryAccessControlFiles()
     {
-        $securedUrl = site_url() . '/wp-content/plugins/versionpress/temp/security-check.txt';
+        $securedUrl = site_url() . '/wp-content/plugins/versionpress/versionpress-nginx.conf';
```

Comment:
The change is correct but maybe we have another small problem here by expecting the `wp-content/plugins` path. Maybe the [`plugins_url` function](https://developer.wordpress.org/reference/functions/plugins_url/) should be used.